src/screens/FeedScreen/Content.js

```diff
@@ -370,12 +372,12 @@ export const styles = StyleSheet.create({
     marginLeft: 8,
     marginRight: 8
   },
-  contentFeed: {
-    flex: 1,
+  contentFeed: (hasComment) => ({
+    flex: !hasComment ? undefined : 1,
     marginTop: 0,
-    height: '100%',
+    height: !hasComment ? dimen.normalizeDimen(355) : '100%',
     width: '100%'
-  },
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic here seems fine, but it might be more readable and maintainable to use ternary operators instead of using the logical NOT operator (`!`). This would make the code easier to understand at first glance.
```diff
-   contentFeed: (hasComment) => ({
-     flex: !hasComment ? undefined : 1,
-     marginTop: 0,
-     height: !hasComment ? dimen.normalizeDimen(355) : '100%',
-     width: '100%'
-   }),
+   contentFeed: (hasComment) => ({
+     flex: hasComment ? 1 : undefined,
+     marginTop: 0,
+     height: hasComment ? '100%' : dimen.normalizeDimen(355),
+     width: '100%'
+   }),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->